builder_data.yaml

```diff
@@ -24,8 +24,8 @@ metrics_url: http://metrics.delivery.puppetlabs.net/overview/metrics
 benchmark: false
 staging_server: weth.delivery.puppetlabs.net
 signing_server: mozart.delivery.puppetlabs.net
-yum_host: 'yum.puppetlabs.com'
-apt_host: 'apt.puppetlabs.com'
+yum_host: 'saturn.delivery.puppetlabs.net'
+apt_host: 'saturn.delivery.puppetlabs.net'
 tar_host: 'downloads.puppetlabs.com'
 msi_host: 'downloads.puppetlabs.com'
 dmg_host: 'downloads.puppetlabs.com'
```

Comment:
are all of the references to downloads also going to need to be updated?

Reply:
as far as I can tell, these values are only used in packaging in the obsolete deploy tasks. I would love to clean all that up, but I think that work is separate from this. 
since the issue that prompted this change is PE-related, these shouldn't be affected.

Reply:
ok. 

Just as an FYI, I think it's just those variables in the release branch that are obsolete, not the tasks. Last time we looked I believe we found those tasks were still in use for things like pl-build-tools, internal-tools, test-tools, etc.

Reply:
oooooh okay, that makes sense, thanks!